DatabaseService/docker_context/endpoints.py

```diff
@@ -28,3 +25,4 @@
 set_sell_trigger_price_endpoint = '/triggers/sell/set/price'
 execute_sell_trigger_endpoint = '/triggers/execute/sell'
 cancel_set_sell_endpoint = '/triggers/sell/cancel'
+add_audit_event = '/audit/event' # ?user_id=string
```

Comment:
similar to previous statement where if user_id was passed as part of the body

Reply:
See response to previous statement.